app/models/mixins/aggregation_mixin.rb

```diff
@@ -8,8 +8,8 @@ module AggregationMixin
     virtual_column :aggregate_vm_cpus,         :type => :integer, :uses => :vms_and_templates
     virtual_column :aggregate_vm_memory,       :type => :integer, :uses => :vms_and_templates
     virtual_column :aggregate_disk_capacity,   :type => :integer, :uses => :hosts
-
-    virtual_column :aggregate_logical_cpus, :type => :integer, :uses => :hosts # Deprecated
+    virtual_column :aggregate_cpu_total_cores, :type => :integer, :uses => :hosts
+    alias_method :aggregate_logical_cpus, :aggregate_cpu_total_cores
```

Comment:
line 37 of this file does do that.
thanks for mentioning this because we want to make sure deprecation notices are there.

There is also another class that adds deprecation notices for these. So I'm thinking that removing this all together may still have this attribute available.